flink-formats/flink-avro-confluent-registry/src/main/resources/META-INF/services/org.apache.flink.table.factories.Factory

```diff
@@ -0,0 +1,16 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+org.apache.flink.formats.avro.registry.confluent.RegistryAvroFormatFactory
```

Comment:
This file does not exist in the release jar (https://mvnrepository.com/artifact/org.apache.flink/flink-avro-confluent-registry/1.11.2)， but some other things. 
Maybe there is any mistake of the maven-shade-plugin config in pom.xml?

Reply:
It is not available, because it will be included in the 1.12 version.

Reply:
Thanks. 
I found it...
I need to use the master branch now...